code/game/turfs/walls/resin.dm

```diff
@@ -205,4 +205,41 @@
 
 /* Hivelord walls, they start off stronger */
 /turf/closed/wall/resin/regenerating/thick
-	max_integrity = 125
+	max_integrity = 250
+
+/turf/closed/wall/resin/regenerating/special
+	name = "you shouldn't see this"
+
+/turf/closed/wall/resin/regenerating/special/attack_alien(mob/living/carbon/xenomorph/X, damage_amount = X.xeno_caste.melee_damage, damage_type = BRUTE, damage_flag = "", effects = TRUE, armor_penetration = X.xeno_caste.melee_ap, isrightclick = FALSE)
+	if(X.status_flags & INCORPOREAL)
+		return
+	X.visible_message(span_xenonotice("\The [X] starts tearing down \the [src]!"), \
+	span_xenonotice("We start to tear down \the [src]."))
+	if(!do_after(X, 1 SECONDS, NONE, X, BUSY_ICON_GENERIC))
+		return
+	if(!istype(src)) // Prevent jumping to other turfs if do_after completes with the wall already gone
+		return
+	X.do_attack_animation(src, ATTACK_EFFECT_CLAW)
+	X.visible_message(span_xenonotice("\The [X] tears down \the [src]!"), \
+	span_xenonotice("We tear down \the [src]."))
```

Comment:
isnt this done on parent

Reply:
Parent includes refunding for Quickbuild points, this subtype removes that. A better way could to have a check in parent itself maybe? Tell me what you want w/ that.